crypto/kzg/src/lib.rs

```diff
@@ -67,13 +72,26 @@ impl Kzg {
 
         Ok(Self {
             trusted_setup: KzgSettings::load_trusted_setup(
-                &trusted_setup.g1_points(),
-                &trusted_setup.g2_points(),
+                &trusted_setup.g1_monomial_points(),
+                &trusted_setup.g1_lagrange_points(),
+                &trusted_setup.g2_monomial_points(),
```

Comment:
Since we changed this from a list of points to a list of flat bytes, I think we should update the names.
```suggestion
                &trusted_setup.g1_monomial_bytes(),
                &trusted_setup.g1_lagrange_bytes(),
                &trusted_setup.g2_monomial_bytes(),
```